doc/mkdocs/mkdocs.yml

```diff
@@ -233,7 +233,7 @@ nav:
       - 'ordered_map': api/ordered_map.md
       - macros:
         - 'Overview': api/macros/index.md
-        - 'JSON_ASSERT(x)': api/macros/json_assert.md
+        - 'JSON_ASSERT': api/macros/json_assert.md
         - 'JSON_CATCH_USER': api/macros/json_throw_user.md
         - 'JSON_DIAGNOSTICS': api/macros/json_diagnostics.md
         - 'JSON_HAS_CPP_11': api/macros/json_has_cpp_11.md
```

Comment:
Should this be named more generically? `JSON_HAS_CPP_<VERSION>`, for example.

Reply:
The names become too long for the sidebar anyway, and I fear that adding shortcuts like these may be also confusing.